.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ defaults:
 jobs:
   unit-test:
     if: github.repository_owner == 'getsentry'
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
Is this necessary?
I do not object to use latest ubuntu, but if it's necessary please use ubuntu 24.04.

Reply:
We need it to get a recent version of Podman on CI servers.

I think keeping it on `ubuntu-latest` is okay as we get warnings when it's gonna change well ahead of time.